core/pom.xml

```diff
@@ -76,4 +79,13 @@
       </build>
     </profile>
   </profiles>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.google.io</groupId>
+      <artifactId>shared</artifactId>
+      <version>${version}</version>
```

Comment:
`${project.version}` is more commonly used I think.. I didn't know there was a plain "version" property!